business_requirement_from_support/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2017 Praxya (https://www.praxya.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement from Support',
+    'category': 'Business Requirements Management',
+    'summary': 'Generate Business Requirements from issues',
+    'version': '8.0.1.0.0',
+    'website': 'https://www.praxya.com/',
+    "author": "Rubén Cabrera Martínez",
+    'depends': [
+        'business_requirement_deliverable_project',
```

Comment:
It made sense to include this one as it is the next logical step to take, generate tasks, but you're right, it is not a dependency, I'll change it to business_requirement.